lib/middleware/index.js

```diff
@@ -59,6 +59,19 @@ function tryParse (json) {
   }
 }
 
+function copyBodyToQuery (req, res, next) {
+  const { query = {}, body = {} } = req
+
+  // prefer token to from query to prevent issues with agol proxies
+  const token = query.token || body.token
+
+  req.query = { ...query, ...body, token }
```

Comment:
Is there any issue with leaving the token on the body? Im not sure what "prevent issues with agol proxies" really means, but I noticed it was previously deleted.

Reply:
shoot, I think you're right, I missed it.

Reply:
Actually, I'm not so sure.  I think it was just more that body.token was always overriding query.token. @mkdir-ito - can you advise?